### PR TITLE
When reloading consul, use 127.0.0.1 as rpc_addr when rpc_addr was set to 0.0.0.0

### PR DESCRIPTION
**Pull request by danielbenzvi**
Solves #220

### CODE REVIEW

File: manifests/reload_service.pp [hunk 1]
```diff
@@ -10,9 +10,18 @@
   # Don't attempt to reload if we're not supposed to be running.
   # This can happen during pre-provisioning of a node.
   if $consul::manage_service == true and $consul::service_ensure == 'running' {
+
+  	# Make sure we don't try to connect to 0.0.0.0, use 127.0.0.1 instead
+  	# This can happen if the consul agent RPC port is bound to 0.0.0.0
+  	if $consul::rpc_addr == '0.0.0.0' {
+      $rpc_addr = '127.0.0.1'
```
Comment (solarkennedy) [inline manifests/reload_service.pp]: Can you clean up the indentation here?

**Comment by solarkennedy** [comment]
Just iterate a bit till the tests pass
https://travis-ci.org/solarkennedy/puppet-consul/jobs/104931189

**Comment by danielbenzvi** [comment]
Done

**Comment by solarkennedy** [comment]
Thanks!

**Pull request merged**